### PR TITLE
update readme with beta and nightly versions

### PR DESCRIPTION
**Pull request by anuragsoni**
Update the read me to reflect the rust beta release and the updated step to install the nightly version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,10 +40,17 @@ catered for.
 The easiest way to install Rust (on Linux and OS X, at least) is to run the rustup
 script. Simply enter the following into your shell:
 
+###Beta Version (Recommended):
 ```
 curl -s https://static.rust-lang.org/rustup.sh | sudo sh
 ```
 
+###To get the Nightly Version:
+(Some rust packages(crates) might not have been updataed to beta just yet)
```
Comment (nrc) [inline README.md]: Typo - updated

Comment (nrc) [inline README.md]: NItty punctuation things: please add a space after `###` and remove the colons (in both cases) and add a full stop after the closing paranthesis. Also a space before `(crates`

File: README.md [hunk 2]
```diff
@@ -40,10 +40,17 @@ catered for.
 The easiest way to install Rust (on Linux and OS X, at least) is to run the rustup
 script. Simply enter the following into your shell:
```
Comment (nrc) [inline README.md]: s/the following/one of the following

**Comment by nrc** [comment]
Thanks for the PR! (I hadn't thought of making the distinction, so it is useful). I asked for a few very minor changes in the comments (sorry it's a bit pernickerty, but it is a documentation project).

**Comment by anuragsoni** [comment]
Updated the PR with changed! I definitely understand about that formatting is important for a documentation project.

**Comment by nrc** [comment]
Awesome, thanks!

**Pull request merged**